setup.cfg

```diff
@@ -56,7 +56,7 @@ dev =
     selenium~=4.7.0
     webdriver-manager~=3.8
 optimade =
-    optimade-client==2022.9.19
+    ipyoptimade~=0.2
```

Comment:
@unkcpz I would first suggest to use version ~=0.1, which should be compatible with current AiiDAlab image. We can then release new AWB version and only then bump it for the next version. WDYT?

Reply:
Yes, it is a good idea. 
I also think about to add a CI workflow to test on aiida-core==2.1. From AWB point of view, the aiida-core is a very base dependencies and deserve to test on different version. Just like we make python version matrix CIs for the package. 

I am think to 
- drop python 3.8 and test on python 3.9/3.10. 
- test on aiida-core 2.1/2.5 with python 3.9 (since we use py39 for our docker image at the moment). But this require to skip some tests with optimade I guess.

Reply:
I am not sure pip will always pick the correct version, especially for Apps that don't declare dependency on aiida-core. As first step I think it is safer to do `~0.1.0`, and then test things vigorously. 

Adding an extra CI workflow for aiida-core versions sounds good. Do you mean that you'd pick an image based on the aiida version tag?

Reply:
Dropping python 3.8 seems fine, although I'd prefer to have that as a separate PR for future reference.

Reply:
> Do you mean that you'd pick an image based on the aiida version tag?

Both, for the notebook test, pick an image on the aiida version tag. 
Also add a unittest CI workflow with pinning aiida-core version to `~2.1`